example/example-metadata.ttl

```diff
@@ -304,8 +304,8 @@
 
 <test-berry> rdf:type dsp-repo:Person . 
 <test-berry> dsp-repo:hasGivenName "Lauren" . 
-<test-berry> dsp-repo:hasFamilyName "Berry" . 
-<test-berry> dsp-repo:hasEmail <lauren.berry@unibas.ch> .
+<test-berry> dsp-repo:hasFamilyName ( "Berry" "Mackenzie" ) . 
```

Comment:
what do the () mean?

Reply:
that's turtle syntax for lists :-)

the good thing about this is that they ensure the order stays the same, whereas having multiple properties will have random order